firestore-bigquery-export/scripts/gen-schema-view/schemas/user_full.json

```diff
@@ -20,11 +20,13 @@
       "fields": [
         {
           "name": "name",
-          "type": "string"
+          "type": "string",
+	  "description": "This is a nested name!"
         }
       ],
       "name": "friends",
-      "type": "map"
+      "type": "map",
+      "description": "Sorry, can't describe a map. What is that?"
```

Comment:
Is this what y'all want in the description for map?

Reply:
Since [maps are interpreted recursively](https://github.com/firebase/extensions/blob/next/firestore-bigquery-export/guides/GENERATE_SCHEMA_VIEWS.md#cloud-firestore-maps), I'm not sure they'll get their own column, so maybe we should take this description out. @IanWyszynski or @fredzqm do you know?


I think this sample query backs up the idea that there won't be a column created: https://github.com/firebase/extensions/blob/9da7a14ef7a053c7e55f2147c99ff706dab8070a/firestore-bigquery-export/scripts/gen-schema-view/src/test/fixtures/sql/nestedMapSchemaChangeLog.txt#L1-L9

Reply:
This is an intentional demonstration of what you've correctly stated above in a testing file.

Reply:
sorry - just to be clear... my comment was about the _phrasing_ used in the description... not on the functionality / technicality of the description.

Reply:
I think the two options are:

1. Take out the description since it isn't used
2. Change to something like `Maps don't need a description because a column is not created for a map. Instead, their fields are added as columns to the resulting dataset.`

Reply:
@jhuleatt I took optiion (2) in the most recent commit.

Reply:
thanks for addressing this :-)